chaco/color_mapper.py

```diff
@@ -405,7 +404,7 @@ def _map(self, X):
 
         """
 
-        if type(X) in [IntType, FloatType]:
+        if type(X) in [int, float]:
```

Comment:
And looking at the comments, we may be able to just remove the whole method if it is in fact no longer used.